watertap/examples/flowsheets/case_studies/electroNP/electroNP_flowsheet.py

```diff
@@ -177,99 +180,229 @@ def build_flowsheet():
-        if "conc_mass_comp[S_ch4]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
-        if "conc_mass_comp[X_su]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
-        if "conc_mass_comp[X_fa]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
-        if "conc_mass_comp[X_c4]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
-        if "conc_mass_comp[X_pro]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
-        if "conc_mass_comp[X_ac]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
-        if "conc_mass_comp[X_h2]" in var.name:
-            iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[S_O2]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[S_N2]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[S_NO3]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_H]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_PAO]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_AUT]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_MeOH]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_MeP]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_TSS]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[S_ch4]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_su]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_fa]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_c4]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_pro]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_ac]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
+        # if "conc_mass_comp[X_h2]" in var.name:
+        #     iscale.set_scaling_factor(var, 1e4)
 
     iscale.calculate_scaling_factors(m)
 
     iscale.set_scaling_factor(m.fs.electroNP.properties_byproduct[0.0].flow_vol, 1e7)
 
+    iscale.set_scaling_factor(m.fs.AD.vapor_phase[0].pressure_sat, 1e-3)
+
+    skip_constraint = [
+        "S_O2",
+        "S_N2",
+        "X_AUT",
+        "X_PHA",
+        "S_I",
+        "S_NO3",
+        "S_Mg",
+        "X_PAO",
+        "S_F",
+        "S_K",
+        "S_A",
+        "X_I",
+        "X_H",
+        "X_PP",
+        "X_S",
+    ]
+
+    for i in skip_constraint:
+        m.fs.electroNP.solute_removal_equation[0.0, "Liq", i].deactivate()
+        m.fs.electroNP.solute_treated_equation[0.0, "Liq", i].deactivate()
+
```

Comment:
@agarciadiego identified these as degenerate. @luohezhiming will run DegeneracyHunter to see if he can identify degenerate constraints and confirm. If confirmed, revise the electronP unit